src/sass/pages/home.scss

```diff
@@ -43,6 +43,7 @@
       border: 0;
       white-space: normal;
       font-size:18px;
+      -webkit-appearance:none;
```

Comment:
we don't need these prefix, we have grunt module to append it and why only webkit?